services/state/schedule/schedule.proto

```diff
@@ -71,19 +71,19 @@ message Schedule {
     /**
      * This schedule's unique ID within the global network state.
      */
-    ScheduleID id = 5;
+    ScheduleID schedule_id = 5;
```

Comment:
Not a fan of `Schedule.scheduleId()` as a pattern.  `Schedule.id()` reads better in almost all cases.

Reply:
I'm agnostic on which is better, but IMO consistency is what's most important and scheduleId() is what's currently in use in more places.

Reply:
This is a good question, generally I'd also omit the foo from fooId so it is foo.id() vs. foo.fooId() in the code -- but I'm not sure if it is too late for that. In all our other protobuf I think we have the redundant name.

Reply:
This PR is where we can change it for state objects.
There are a few with ID and a few with thingID, so making them consistent at just ID for state objects is doable now, but would be very hard to do later.

Reply:
I agree with `id` for your _own_ id, `foo_id` for someone else's.  SImiliar to how you'd name primary keys vs foreign keys in a table.